src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -890,4 +890,8 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
         this.action = SideNavAction.BESLUIT_WIJZIGEN;
         this.actionsSidenav.open();
     }
+
+    private isIntake(zaak: Zaak): boolean {
```

Comment:
Een meer Object Oriented aanpak is om deze methode aan te maken op de Zaak typescript class zodat je kunt vragen: zaak.isIntake(). Daar komt dan deze check te staan. Maak van de String 'Intake' een final static properrty in de Zaak class.

Echter gezien het commentaar hierboveb is dit wellicht ook niet de juiste opossing. Denk er eens goed over na. Overleg eventueel met mij.

Reply:
Via Teams besproken en geimplementeerd